### PR TITLE
Migrated the site to Vanilla V1

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Migrated the site to Vanilla V1

## QA
- Pull down code
- Run `/run`
- Check the site looks ok 

## Issue / Card
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/5 https://github.com/canonical-websites/cloud-init.io/issues/50 https://github.com/canonical-websites/cloud-init.io/issues/61

## Demo
http://cloud-init-v1.demo.haus/

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-
-        <p class="footer__text">&copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
-
-        <ul class="footer__links">
-            <li class="footer__list-item"><a href="http://www.ubuntu.com//legal">Legal information</a><span class="footer__dot"> &#183;</span></li>
-            <li class="footer__list-item"><a href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a></li>
+<footer class="p-footer">
+  <div class="row">
+    <div class="col-12">
+      &copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
```
Comment (richmccartney) [inline _includes/footer.html]: Is there a way that we can make the year a bit more dynamic?

File: _includes/footer.html [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+      <nav class="p-footer__nav">
+        <ul class="p-footer__links">
+          <li class="p-footer__item">
+            <a class="p-footer__link" href="http://www.ubuntu.com//legal">Legal information</a>
+          </li>
+          <li class="p-footer__item">
+            <a class="p-footer__link" href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a>
```
Comment (richmccartney) [inline _includes/footer.html]: Do we need to add ```external``` to these links as well as ```aria-label``` to say its going to an external site?

File: _includes/footer.html [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
-            <li class="footer__list-item"><a href="http://www.ubuntu.com//legal">Legal information</a><span class="footer__dot"> &#183;</span></li>
-            <li class="footer__list-item"><a href="https://github.com/ubuntudesign/cloud-init/issues/new">Report a bug on this site</a></li>
+<footer class="p-footer">
+  <div class="row">
+    <div class="col-12">
+      &copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+      <nav class="p-footer__nav">
+        <ul class="p-footer__links">
+          <li class="p-footer__item">
+            <a class="p-footer__link" href="http://www.ubuntu.com//legal">Legal information</a>
```
Comment (richmccartney) [inline _includes/footer.html]: Link is broken

File: index.html [hunk 4]
```diff
@@ -2,295 +2,392 @@
...
-            <a href="https://www.youtube.com/watch?v=-zL3BdbKyGY" class="hero__cta button--primary mediabox">
-                Watch tutorial
-            </a>
+<section class="p-strip p-strip--accent is-dark">
+  <div class="row">
+    <div class="col-12 suffix-4">
+      <div class="p-heading-icon">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img"
+            src="https://assets.ubuntu.com/v1/4fd55d51-picto-corepower-orange.svg"
```
Comment (richmccartney) [inline index.html]: Should we explain what the image is?

File: index.html [hunk 5]
```diff
@@ -2,295 +2,392 @@
...
+            alt="">
+          <h3 class="p-heading-icon__title">cloud-init</h3>
         </div>
+      </div>
+      <h1>The standard for customising cloud instances</h1>
+      <a class="p-button--brand"
```
Comment (richmccartney) [inline index.html]: Lets give this an ```aria-label``` explaining its going to youtube to watch a video, and what the content is about.

**Comment by richmccartney** [comment]
@anthonydillon I don't know if you generally want to go through all the ```.p-link--external``` elements and add ```aria-label``` to explain in a simple way that the link will take a user away to an external page.

File: _includes/footer.html (hunk 1 above)
Comment (anthonydillon) [inline _includes/footer.html]: Good point, done.

File: _includes/footer.html (hunk 3 above)
Comment (anthonydillon) [inline _includes/footer.html]: Good spot.

File: _includes/footer.html (hunk 2 above)
Comment (anthonydillon) [inline _includes/footer.html]: Done.

File: index.html (hunk 4 above)
Comment (anthonydillon) [inline index.html]: It's only decoration. Doesn't add value to the content.

File: index.html (hunk 5 above)
Comment (anthonydillon) [inline index.html]: Done.

**Comment by anthonydillon** [comment]
Thanks @richmccartney all points addressed.

**Comment by yaili** [comment]
Design review:
- [ ] Let's go back to blue links pls, sorry
- [ ] Border missing above "Source code"

**Comment by anthonydillon** [comment]
@yaili all done

**Comment by richmccartney** [comment]
@anthonydillon approved 👍

**Pull request merged**